CNAME

```diff
@@ -1 +1 @@
-git.youphptube.com
\ No newline at end of file
+www.sfrancis.ca
```

Comment:
@DanielnetoDotCom @sfrancis2017

Is this change intentional or accidental (the `CNAME` change)?